tests/test_run_feature_vector.py

```diff
@@ -7,22 +7,17 @@
 import pytest
 from dictdiffer import diff
 
+path_to_current_file = os.path.realpath(__file__)
+current_directory = os.path.split(path_to_current_file)[0]
+
+TEST_OUTPUT_DIR = os.path.join(current_directory, "data/feature_vector")
```

Comment:
there is no point using os.path.join for cross-platform compatability then hardcoding a path separator. This should be:
```
os.path.join(current_directory, "data", "feature_vector")
```

Reply:
changed, thanks.